docs/csharp/tour-of-csharp/program-building-blocks.md

```diff
@@ -47,13 +51,13 @@ In the following example, each instance of the `Color` class has a separate copy
 
 :::code language="csharp" source="./snippets/shared/ClassesObjects.cs" ID="ColorClassDefinition":::
 
-As shown in the previous example, *read-only fields* may be declared with a `readonly` modifier. Assignment to a read-only field can only occur as part of the field’s declaration or in a constructor in the same class.
+As shown in the previous example, *read-only fields* may be declared with a `readonly` modifier. Assignment to a read-only field can only occur as part of the field's declaration or in a constructor in the same class.
 
 ## Methods
```

Comment:
Should this be nested under "Members"?

Reply:
Logically it should, but as a practical matter it would make a very big h2 section with h3s into an h3 with a bunch of h4s, and then **Fields** and **Other function members** would have to be demoted too, so I think keeping it as an h2 makes sense.